backbone.stickit.js

```diff
@@ -174,7 +174,11 @@
 
             // Don't update the model if false is returned from the `updateModel` configuration.
             var currentVal = evaluateBoolean(config.updateModel, val, event, config);
-            if (currentVal) setAttr(model, modelAttr, val, options, config);
+            if (isPromise(currentVal)) {
+              currentVal.then(function(){
+                setAttr(model, modelAttr, val, options, config);
```

Comment:
Also it looks like the `currentVal` is not actually using a return value from the promise. Shouldn't `currentVal` return a `val`?